cmd/kubeadm-nspawn/kubeadm-nspawn.go

```diff
@@ -162,7 +198,7 @@ func newKubeadmNspawnCommand() *cobra.Command {
 		},
 	}
 	cmd.Flags().BoolVarP(&printVersion, "version", "V", false, "output version information")
-	cmd.PersistentFlags().StringVarP(&k8srelease, "kubernetes-version", "k", "", "Kubernetes version to spawn")
+	cmd.PersistentFlags().StringVarP(&k8srelease, "kubernetes-version", "k", k8sStableVersion, "Kubernetes version to spawn")
```

Comment:
Maybe we can have a special word to signal that we want to use local kubernetes. Something like `--kubernetes-version=dev` or setting the flag but passing an empty value.